.editorconfig

```diff
@@ -0,0 +1,18 @@
+# https://EditorConfig.org
+root = true
+
+[*.yml]
+indent_style = space
+indent_size = 2
+end_of_line = lf
```

Comment:
Move the duplicate definitions under [*]?

Reply:
done